<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #444444;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="solutionButton">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>310</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
background-color: #3071E7;
color: white;</string>
    </property>
    <property name="text">
     <string>Get solution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>375</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
background-color: #3071E7;
color: white;</string>
    </property>
    <property name="text">
     <string>Change task</string>
    </property>
   </widget>
   <widget class="QLabel" name="taskLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="solutionLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color: white;</string>
    </property>
    <property name="text">
     <string>Solution</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="param1Edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Parameter 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="param2Edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Parameter 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="param3Edit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>340</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255, 30);
border-radius: 7px;
color: white;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Parameter 3</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
